src/app/screens/connect/btcSelectAddressScreen/index.tsx

```diff
@@ -0,0 +1,292 @@
+import BitcoinIcon from '@assets/img/dashboard/bitcoin_icon.svg';
+import stxIcon from '@assets/img/ledger/stx_icon.svg';
+import OrdinalsIcon from '@assets/img/nftDashboard/white_ordinals_icon.svg';
+import ActionButton from '@components/button';
+import SelectAccount from '@components/selectAccount';
+import useBtcAddressRequest from '@hooks/useBtcAddressRequest';
+import useWalletSelector from '@hooks/useWalletSelector';
+import { Check } from '@phosphor-icons/react';
+import { animated, useSpring } from '@react-spring/web';
+import { StickyHorizontalSplitButtonContainer } from '@ui-library/common.styled';
+import { getTruncatedAddress } from '@utils/helper';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { AddressPurpose } from 'sats-connect';
+import styled, { useTheme } from 'styled-components';
+import { getAppIconFromWebManifest } from './helper';
+
+const TitleContainer = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'hidden',
+  marginLeft: 30,
+  marginRight: 30,
+  marginTop: 48,
+});
+
+const OuterContainer = styled(animated.div)({
+  display: 'flex',
+  flexDirection: 'column',
+  marginLeft: 16,
+  marginRight: 16,
+});
+
+const AddressBoxContainer = styled.div((props) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%',
+  marginTop: props.theme.spacing(12),
+}));
+
+const AddressBox = styled.div((props) => ({
+  width: 328,
+  height: 66,
+  padding: props.theme.spacing(10),
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  backgroundColor: props.theme.colors.elevation6_800,
+  marginBottom: 1,
+  ':first-of-type': {
+    borderTopLeftRadius: props.theme.radius(2),
+    borderTopRightRadius: props.theme.radius(2),
+  },
+  ':last-of-type': {
+    borderBottomLeftRadius: props.theme.radius(2),
+    borderBottomRightRadius: props.theme.radius(2),
+  },
+}));
+
+const AddressContainer = styled.div({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const TopImage = styled.img({
+  maxHeight: 32,
+  maxWidth: 32,
+});
+
+const Title = styled.h1((props) => ({
+  ...props.theme.typography.body_bold_l,
+  color: props.theme.colors.white_0,
+  marginTop: 12,
+}));
+
+const DapURL = styled.h2((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_400,
+  marginTop: props.theme.spacing(2),
+  textAlign: 'center',
+}));
+
+const AddressImage = styled.img({
+  width: 24,
+  height: 24,
+  marginRight: 8,
+});
+
+const AddressTextTitle = styled.h2((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_200,
+  textAlign: 'center',
+}));
+
+const TruncatedAddress = styled.h3((props) => ({
```

Comment:
thank you for using semantic html!

nit: most body_medium_m typography should be a <p>